readme.md

```diff
@@ -34,16 +34,16 @@ gulp.task('default', function () {
 
 ##### ui
 
-Type: `string`  
```

Comment:
All the double spaces in this API doc are explicitly added so you shouldn't remove them.